.github/workflows/issue-create.yml

```diff
@@ -0,0 +1,37 @@
+on:
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+#  push:
+#    paths:
+#      - 'data/**'
+#    branches:
+#      - master
+name: Git Issue Create On EU Data Update
```

Comment:
Since this is a workflow to handle any publish activities, I think it would be good to use a more generic name like QPP EU Data Publish, like the QPP Measures Data repo does. The same applies to the file name, it would be better to call it publish.yml